index.html

```diff
@@ -115,6 +115,31 @@ <h2>A project for first-time contributions</h2>
         <!-- Keep one line of space above and below your card -->
         <!-- ========= Paste YOUR CARD directly BELOW this line ========= -->
 
+        <div class="card">

```

Comment:
Hi, your card is missing the opening and the ending comment. Card Start and card end. Check the template as a guide.
Thanks